.github/workflows/ci-fast.yml

```
@@ -82,29 +82,79 @@ jobs:
       python-version: ${{ matrix.python-version }}
       test_environment: ${{ matrix.test_environment }}
     secrets: inherit
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write  # Needed for pushing to GHCR
+    steps:
+      - uses: actions/checkout@v4.1.1
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v1
```

actionlint

the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)